### DISCUSSION TITLE
Privacy Manifest

### DISCUSSION THREAD

**Discussion by gadellaccio** [siteline/SwiftUI-Introspect#398] [Q&A] [answered]
Hello, 

I have a question regarding the new privacy manifest files that Apple is going to start requiring as of spring 2024. Does the swiftui-introspect team plan on adding a privacy manifest to support this new iOS requirement? Apple’s full explanation of the matter can be found [here](https://developer.apple.com/news/?id=r1henawx) 
Thanks for taking the time to look into this, any additional context or potential dates for support privacy manifests would be helpful.

**Comment by davdroman** [comment, marked as answer]
See https://github.com/pointfreeco/swift-composable-architecture/discussions/2559#discussioncomment-8005185

I don't believe SwiftUIIntrospect leverages any APIs that require a privacy manifest, and it definitely doesn't collect any user data. Correct me if I'm wrong though.

**Reply by gadellaccio** [reply]
Ok, great. Thank you very much for your reply!

**Comment by frankois1234** [comment]
Sorry, but even the SDK collect no data, the sdk must contain the manifest which say explicitly it doesn't collect data. Even RxSwift must contain the maniest

**Reply by gadellaccio** [reply]
Thank you for that clarification @frankois1234. 
Tagging @davdroman for visibility.
